### PR TITLE
Add 404 page

### PR DESCRIPTION
**Pull request by benslv**

### CODE REVIEW

File: src/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <Layout title="Page Not Found">
+      <div className="container">
+        <div className="row my-5">
+          <div className="col-12">
+            <h1>Page not found</h1>
+            <p>
+              Oops! That page doesn&apos;t exist. Double check the URL to make sure it&apos;s
+              correct, otherwise you can return to the homepage <TextLink to="/">here</TextLink>.
+            </p>
```
Comment (Sciencentistguy) [inline src/pages/404.js:16]: Maybe put some sort of graphic on image on this page? It looks a bit bare imo.

**Review by Sciencentistguy — APPROVED** [review]
I've left a comment, but otherwise LGTM.

File: src/pages/404.js (hunk 1 above)
Comment (benslv) [reply src/pages/404.js:16]: What do you suggest?

Comment (Sciencentistguy) [reply src/pages/404.js:16]: I'm not sure. Maybe the animation that's on the homepage?

**Pull request merged**